File: ProblemSet/734A_Anton and Danik/C/solution.c
// Date : 28-Sep-2020 By Jatin Sharma

#include<stdio.h>
#include<string.h>

int main(){
    int games, anton = 0 , danik = 0;
    scanf("%d", &games);
    char win[games];
    scanf("%s", win);
    for (int i = 0; i < strlen(win); i++)
    {
        if (win[i] == 'A'){
            anton++;
        }
        else if (win[i] == 'D')
        {
            danik++;
        }
    }

    if (anton > danik)
    {
        printf("Anton");
    } else if (anton < danik)
    {
        printf("Danik");
    } else if (anton==danik) {
        printf("Friendship");
    }
    
    return 0;
    

}

File: ProblemSet/734A_Anton and Danik/Python/solution.py
#  Date : 28-Sep-2020 By Jatin Sharma

games = int(input())
win = input()
anton = 0
danik = 0

for i in range(len(win)):
    if win[i]=="A":
        anton+=1
    elif win[i] == "D":
        danik+=1


if anton > danik:
    print("Anton")
elif anton < danik:
    print("Danik")
elif anton == danik:
    print("Friendship")